cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -3297,17 +3291,21 @@ pub(crate) fn define(
             "breduce",
             r#"
         Convert `x` to a smaller boolean type in the platform-defined way.
```

Comment:
I don't think that `in the platform-defined way` is a useful specification, but the operation is equivalent to the integer one, so you can just repeat the same statement here, i.e. `by discarding the most significant bits`.